src/els_document_highlight_provider.erl

```diff
@@ -52,21 +52,22 @@ find_highlights(Uri, #{ kind := Kind
           {M, F, A} -> {M, F, A};
           V         -> {els_uri:module(Uri), V, 0}
         end,
-  case els_db:find_multi(references, Key) of
-    {error, not_found} ->
+  case els_dt_references:find_by_id(Key) of
+    {ok, []} ->
       null;
     {ok, Refs} ->
-      R = [document_highlight(R) ||
-          {_, #{uri := U, range := R}} <- ordsets:to_list(Refs)
-          , Uri == U ],
+      R = [ document_highlight(R) ||
+            #{uri := U, range := R} <- ordsets:to_list(Refs)
+              , Uri == U
+          ],
       R
   end;
 find_highlights(_Uri, _POI) ->
   null.
 
 -spec document_highlight(poi_range()) -> map().
 document_highlight(Range) ->
-    %% TODO:AZ:range as displayed ends one col too early
+  %% TODO:AZ:range as displayed ends one col too early
```

Comment:
This was probably fixed already by [this change](https://github.com/erlang-ls/erlang_ls/pull/280/files#diff-3f144025dae38c6a860fe792f9f86294R26).

Reply:
Removed.